stdlib/@tests/test_cases/ctypes/check_CDLL.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
+
+import ctypes
+from pathlib import Path
+from typing import Type
+from typing_extensions import assert_type
+
+assert_type(ctypes.CDLL(None), Type[ctypes.CDLL])
```

Comment:
You can probably remove the `Type[]` part; `CDLL()` should return an instance, not a type.

However, I'm not sure this merits a test case at all, since the stub seems straightforward. We tend to write tests only for more intricate cases, such as complex overloads or generics.